src/Microsoft.Diagnostics.Monitoring.RestServer/CorsConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    public class CorsConfiguration
+    {
+        public string AllowedOrigins { get; set; }
+
+        public string[] GetOrigins() => AllowedOrigins?.Split(';');
```

Comment:
The standard split character typically used to specify list of CORS origins is usually a space. I personally prefer a ';' so I would just keep this as-is.